helm/auto-deployment/bold-bi-auto-deployment-using-helm.md

```diff
@@ -0,0 +1,439 @@
+       namespace
+      </td>
+      <td>
+       The namespace in which the Bold BI resources will be dpleoyed in the kubernetes cluster.<br/>
+       The default namespace is <i>bold-services</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       appBaseUrl *
+      </td>
+      <td>
+       Domain with http/https protocol.
+       <br/>
+       Ex: `http://example.com`, `https://example.com`
+       <br/>
+       Follow the <a href='configuration.md#ssl-configuration'>SSL Configuration</a> to configure SSL certificate for https protocol.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       optionalLibs
+      </td>
+      <td>
+       These are the client libraries used in Bold BI by default.<br/>
+       '<i>mongodb,mysql,influxdb,snowflake,oracle,google,clickhouse</i>'<br/>
+       Please refer to <a href='docs/configuration.md#client-libraries'>Optional Client Libraries</a> section to know more.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using.<br/>
+       The supported values are '<i>gke,eks and aks</i>'
+       Please refer to <a href='configuration.md#cluster-provider'>Cluster Provider</a> section to know more.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume*
+      </td>
+      <td>
+       This is a file storage information to store the shared folders for application usage.
+       Please refer to <a href='configuration.md#persistent-volume'>this</a> section to know more on how to set Persistant Volumes for Bold BI.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       loadBalancer
+      </td>
+      <td>
+       Currently we have provided support for Nginx and Istio as Load Balancers in Bold BI. Please refer to <a href='docs/configuration.md#load-balancing'>this</a> section for configuring Load balancer for Bold BI.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling
+      </td>
+      <td>
+       By default autoscaling is enabled in Bold BI. Please refer to <a href='configuration.md#auto-scaling'>this</a> section to configure autoscaling in Bold BI.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       bingMapWidget
+      </td>
+      <td>
+       Please refer to <a href='configuration.md#bing-map-widget'>this</a> section to configure Bing Map Widget in Bold BI.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       customLocalePath
+      </td>
+      <td>
+       Custom locale file path for Localization.
+       Please refer to <a href='configuration.md#custom-locale-path'>this</a> section to configure Custom Locale Path in Bold BI.
+      </td>
+    </tr>
+    </table>
+<br/>
+
+> **Note:** Items marked with `*` are mandatory fields in values.yaml
+
+## Environment variables for configuring Branding in backend
+
+This following environment variables helps you to deploy Bold BI in Kubernetes without manually activating licensing and configuring startup from the browser.
```

Comment:
The following environment variables allow you to deploy Bold BI in Kubernetes without manually activating licensing and configuring startup from the browser.

Reply:
concern addressed